starter-code/pizza.js

```diff
@@ -1 +1,100 @@
 // Write your Pizza Builder JavaScript in this file.
+
+$ (document).ready(function(){
+
+
+  $(".btn-sauce").removeClass("active");
+  $('.price li:contains(sauce)').toggle();
+  $(".btn-crust").removeClass("active");
+  $('.price li:contains(crust)').toggle();
+
+  function operation(){
+
+    var ingredientes=10;
+
+    if ($(".btn-pepperonni").hasClass("active")) {
+       ingredientes += 1;
+    }
+    if ($(".btn-mushrooms").hasClass("active")) {
+       ingredientes += 1;
+    }
+    if ($(".btn-green-peppers").hasClass("active")){
+       ingredientes += 1;
+    }
+    if ($(".btn-sauce").hasClass("active")) {
+       ingredientes += 3;
+    }
+    if ($(".btn-crust").hasClass("active")){
+       ingredientes += 5;
+    }
+
+    precioTotal= '$' + ingredientes;
+
+    $('.price strong').html(precioTotal);
+
+  }
+    $('.price strong').ready(operation());
+
+        $('.btn-pepperonni').click(function(){
+          $('.pep').toggle();
+          $('.price li:contains(pepperonni)').toggle();
+
+          if ($(".btn-pepperonni").hasClass("active")) {
```

Comment:
For the next time, insted of checking the class and adding and removing you can simply use the method toggleClass()